core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -189,6 +216,34 @@ export class TranslationModalComponent {
     };
   }
 
+  ngAfterViewInit(): void {
+    this.computePanelOverflowState();
+  }
+
+  ngAfterViewChecked(): void {
+    this.computeTranslationEditorOverflowState();
+  }
+
+  computeTranslationEditorOverflowState(): void {
+    setTimeout(() => {
+      const windowHeight = this.wds.getHeight();
+      let heightLimit = windowHeight * (this.cutoff_height) / 100;
+
+      this.isTranslationOverflowing = (
+        this.translationContainer?.nativeElement.offsetHeight >=
+        heightLimit
+      );
+    }, 500);
```

Comment:
How was this value chosen?

Reply:
This value was chosen mostly by trial and error. A delay of 500ms ensures that the content has had enough time to load before the height/overflow is calculated.

Reply:
This delay in the `computeTranslationEditorOverflowState` function is unnecessary, however it is needed in the `computePanelOverflowState` function because the 'content overflow' only needs to be calculated once and adding it to the `ngAfterViewChecked' hook would lead to unnecessary processing which could potentially reduce performance on some devices.
The value for the delay was chosen pretty much by trial and error to ensure that the content has had enough time to load before the overflow was calculated. Values smaller than 500ms also work but they sometimes give inconsistent results.